hotload.py

```diff
@@ -46,6 +46,21 @@ def _file_changed(f):
 def _all_file_changes(filepaths):
     return {path: _file_changed(path) for path in filepaths}
 
+def _changed_modules(new_changed, last_changed):
+    list = []
```

Comment:
`list` shadows the `list` class.

Perhaps call it `changed_modules`?

Reply:
👍